src/Expensify.js

```diff
@@ -64,6 +65,8 @@ class Expensify extends PureComponent {
     constructor(props) {
         super(props);
 
+        setTimeout(() => this.reportBootSplashStatus(), 30 * 1000);
```

Comment:
Would this be better suited in `componentDidMount()` instead? I don't see a huge reason for it to be in the constructor.